assets/js/load_more.js

```diff
@@ -0,0 +1,36 @@
+jQuery(document).ready(function () {
+    initLoadMore();
+})
+
+function initLoadMore() {
+    jQuery('.jtf-load-more').on('click', function (e) {
+        e.preventDefault();
+        var $loadmore = jQuery(this),
+            link = $loadmore.attr('href'),
+            selector = $loadmore.attr('data-selector'),
+            container = $loadmore.attr('data-container'),
+            attribute = $loadmore.attr('data-attribute'),
+            cont = ( (attribute == 'class' ) ? '.' : '#') + selector;
+        jQuery.post(
+            link,
+            {
+                'jtf-selector': selector,
+                'jtf-container': container,
+                'jtf-attribute': attribute,
+                'jtf_load_more': true
```

Comment:
would be easier for server side to send array as:
jtf: {selector: ..., container: ..., ... }

in this case you don't duplicate jtf prefix lot of times on backend